src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmissingwhitespaceafterasterisk/InputJavadocMissingWhitespaceAfterAsterisk.java

```diff
@@ -0,0 +1,53 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocmissingwhitespaceafterasterisk;
+class InputJavadocMissingWhitespaceAfterAsterisk
+{
+    /****
+     *
+     * This is ok.
+     ****/
+    void foo() {}
+
+    /**
+     *  @see See somthing.
+     */
+    void foo1() {}
+
+    /**
+     *This is not ok.
+     **/
+    void foo2() {}
+
+    /**
+     * This is not ok.
+     *@see Something
+     **/
+    void foo3() {}
+
+    /**
+     * @see
+     *This is not ok.
+     **/
+    void foo4() {}
+
+    /**
+     * <pre>
+     *This is not ok.
+     * </pre>
+     */
+    void foo5() {}
+
+    /******** This is ok. *******/
+    void bar() {}
+
+    /** @see Something */
+    void bar1() {}
+
+    /**@see Something*/
+    void bar2() {}
+
+    /***/
+    void bar4() {}
+
+    /** This is ok. **/
```

Comment:
Please add a test case for
`/** *violation */`
A single-line Javadoc with leading asterisk. Yes, it is weird, but it is legal.

Reply:
done